src/grt/src/Grid.cpp

```diff
@@ -136,6 +136,40 @@ void Grid::getBlockedTiles(const odb::Rect& obstruction,
   last_tile_bds = odb::Rect(ll_last_tile, ur_last_tile);
 }
 
+interval<int>::type Grid::computeTileReduceInterval(const odb::Rect& obs,
+                            const odb::Rect& tile,
+                            int track_space,
+                            bool first,
+                            odb::dbTechLayerDir direction)
+{
+  int start_point, end_point;
+  if (direction == odb::dbTechLayerDir::VERTICAL) {
+    if (obs.xMin() >= tile.xMin() && obs.xMax() <= tile.xMax()) {
+      start_point = obs.xMin();
+      end_point = obs.xMax();
+    } else if (first) {
+      start_point = obs.xMin();
+      end_point = tile.xMax();
+    } else {
+      start_point = tile.xMin();
+      end_point = obs.xMax();
```

Comment:
In the first tile wouldn't you want to start from the tile.xMin?

Reply:
Not really. If I understand correctly what Luis did, we want the interval blocked by the obs. The figure below shows why using the obs.xMin is correct:
<img width="649" alt="image" src="https://user-images.githubusercontent.com/43450810/179292761-65f1c2f3-c5cd-459d-97fd-011563f7be0a.png">

Reply:
got it